common/templates/context_funcs.go

```diff
@@ -1075,9 +1075,9 @@ func (c *Context) tmplCreateThread(channel, msgID, name interface{}, optionals .
 				return nil, errors.New("createThread 'private' must be a boolean")
 			}
 		case 1:
-			duration := tmplToInt(opt)
-			if duration < 60 || duration > 10080 {
-				return nil, errors.New("createThread 'auto_archive_duration' must be and integer between 60 and 10080")
+			start.AutoArchiveDuration = tmplToInt(opt)
+			if start.AutoArchiveDuration < 60 || start.AutoArchiveDuration > 10080 {
```

Comment:
Shouldn't this check exactly `60`, `1440`, `4320`, and `10080`, as stated in the [thread metadata docs](https://discord.com/developers/docs/resources/channel#thread-metadata-object)?
Currently giving any other value won't create the thread, with integers between 60 and 10080 failing silently.

Reply:
Done this in `191fa78`, I've also added a designated type and constants for auto archive duration for readability in `283301f`.